health return type not compatible `FastAPI.add_api_route`

The FastAPI method add_api_route takes in `Callable[..., Coroutine[Any, Any, Response]]` but the health method for this library returns `Callable[..., Awaitable[JSONResponse]]` which raises an error in MyPy.

```python
# test.py
from fastapi import FastAPI
from fastapi_health import health

def is_database_online() -> bool:
    return True

app = FastAPI()
app.add_api_route(
    "/health",
    health([is_database_online]),
)
```

```
test.py:12: error: Argument 2 to "add_api_route" of "FastAPI" has incompatible type "Callable[..., Awaitable[JSONResponse]]"; expected "Callable[..., Coroutine[Any, Any, Response]]"  [arg-type]
Found 1 error in 1 file (checked 1 source file)
```

Changing the method signature to `Callable[..., Coroutine[Any, Any, JSONResponse]]` would fix these warnings.

MyPy version: 0.991
FastAPI version: 0.89.1

--- a/fastapi_health/route.py
+++ b/fastapi_health/route.py
@@ -1,6 +1,6 @@
 from inspect import Parameter, Signature
 from typing import TypeVar
-from typing import Any, Awaitable, Callable, Dict, List, Union
+from typing import Any, Awaitable, Callable, Coroutine, Dict, List, Union
 
 from fastapi import Depends
 from fastapi.encoders import jsonable_encoder
@@ -32,7 +32,7 @@ def health(
     failure_handler: Callable[..., Awaitable[dict]] = default_handler,
     success_status: int = 200,
     failure_status: int = 503,
-) -> Callable[..., Awaitable[JSONResponse]]:
+) -> Callable[..., Coroutine[None, None, JSONResponse]]:
     """Create a health check route.
 
     Args:

